dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNDataInterfaceIIDM.h

```diff
@@ -177,6 +183,14 @@ class DataInterfaceIIDM : public DataInterface {
    */
   boost::shared_ptr<ComponentInterface>& findComponent(const std::string& id);
 
+  /**
+   * @brief find a component thanks to its id
+   * @param id : id of the component to find
+   *
+   * @return instance of component interface found
+   */
+  const boost::shared_ptr<ComponentInterface>& findComponent(const std::string& id) const;
```

Comment:
For pointer it is better to use a copy for me (same performance cost as we copy a c pointer

Reply:
I'd keep this as it is to have symmetry with respect to other declaration of the method